docs/Introduction.md

```diff
@@ -1,11 +1,15 @@
 # AMWA MS-05-02 NMOS Control Framework \[Work In Progress\]
 
 The NMOS Control framework is a collection of class and datatype models and supporting documentation.
-It is aimed to serve as the foundation for building devices, components and features in a standard and reusable fashion.
+It is aimed to serve as the foundation for building devices, components and control and monitoring features in a standard and reusable fashion.
 
 This specification relies on previous familiarity with the following related specifications:
 
 - [MS-05-01 NMOS Control Architecture](https://specs.amwa.tv/ms-05-01)
 - [IS-12 NMOS Control Protocol](https://specs.amwa.tv/is-12)
 
-The aggregated collection of control class and datatype models is published in the [Framework](Framework.md).
+The aggregated collection of control class and datatype models is published in the [Framework](Framework.md) with specific functionality published separately as [Feature sets](Feature%20sets.md).
+
+Models of both classes and datatypes published either in the [Framework](Framework.md) or in one of the [Feature sets](Feature%20sets.md) are deemed as `standard` models and MUST have their names prefixed with `"Nc"`.
```

Comment:
Suggested simplification
```suggestion
Models of both classes and datatypes published either in the [Framework](Framework.md) or in one of the [Feature sets](Feature%20sets.md) are `standard` models and MUST have their names prefixed with `"Nc"`.
```

Reply:
Why is the word standard in backticks, rather than single or double quotes (standard but not really standard 😉)?

Can vendors publish feature sets into the register? If so do they have to be prefixed with `Nc` too or not?

Reply:
Vendors cannot publish to the register directly. They would need to "elevate" any proposed models at which point they would become standard models.